lib/floe/workflow_base.rb

```diff
@@ -77,6 +77,11 @@ def output(context)
       context.output.to_json if end?(context)
     end
 
+    # overrides ValidationMixin#workflow_state?
+    def workflow_state?(field_value, _workflow)
+      @payload["States"].include?(field_value)
```

Comment:
Is it possible to get here and States not exist? - I'm not sure if the presence of States is validated first or not. If not, 
```suggestion
      @payload["States"]?.include?(field_value)
```